quill-core/src/test/scala/io/getquill/quotation/QuotationSpec.scala

```diff
@@ -847,17 +849,18 @@ class QuotationSpec extends Spec {
         l.encoder mustEqual intEncoder
       }
       "property" in {
-        case class Test(a: String)
-        val t = Test("a")
+        case class TestEntity(a: String)
+        val t = TestEntity("a")
         val q = quote(lift(t.a))
 
         val l = q.liftings.`t.a`
         l.value mustEqual t.a
         l.encoder mustEqual stringEncoder
       }
       "abritrary" in {
-        val q = quote(lift(String.valueOf(1)))
-        q.liftings.`java.this.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
+        class A { def x = 1 }
+        val q = quote(lift(new A().x))
+        q.liftings.`new A().x`.value mustEqual new A().x
```

Comment:
@fwbrasil I hope this work for such test case, since using static calls as before is prohibit fr cross compilation